index.js

```diff
@@ -1,14 +1,30 @@
-export default function mathClamp(number, {minimum, maximum}) {
-	if (minimum > maximum) {
-		throw new RangeError('`minimum` should be lower than `maximum`');
+export default function mathClamp(number, {min, minimum, max, maximum}) {
+	if (minimum !== undefined) {
+		if (min === undefined) {
+			min = minimum;
+		} else {
+			throw new TypeError('`min` and `minimum` are mutually exclusive');
```

Comment:
I don't think this is nessecary. It should just be an alias. Just make `min` override `minimum`.